pkgs/applications/networking/python-eduvpn-client/default.nix

```diff
@@ -0,0 +1,45 @@
+{ pkgs, stdenv, fetchFromGitHub, pythonPackages, gtk3, libnotify
+, gobjectIntrospection, networkmanager_openvpn, wrapGAppsHook
+, gdk_pixbuf }:
+
+with pythonPackages;
+
+buildPythonApplication rec {
+  pname = "eduvpn-client";
+  version = "1.0rc11";
+
+  src = fetchFromGitHub {
+    owner = "eduvpn";
+    repo = "python-${pname}";
+    rev = "${version}";
+    sha256 = "1lw6ip2d511ppfl40bfnqadbmjw73ml5m2fabrv6h778icv29nkg";
+  };
+
+  prePatch = ''
+    substituteInPlace eduvpn/util.py \
+      --replace "/usr/local" "$out"
+  '';
+
+  checkInputs = [
+    pytest mock
+  ];
+
+  nativeBuildInputs = [ wrapGAppsHook pytestrunner ];
+
+  propagatedBuildInputs = [
+    gtk3 libnotify dbus-python pygobject3 pynacl configparser
+    requests requests_oauthlib future mock repoze_lru
+    dateutil gobjectIntrospection networkmanager_openvpn six
+    gdk_pixbuf
+  ];
```

Comment:
It would be nice if you split these up into python and non-python dependencies. Just add a comment such as `# non-python dependencies`.
Alternatively, get rid of the single `with pythonPackages;` above and split the list into two list where one has `with pythonPackages;`. This has the advantage of avoiding problems when a global package has the same name as a python package.

Reply:
I could not get the latter to work (I got "syntax error, unexpected WITH") so I opted for the first.

Reply:
You'd have to use `(with pythonPackages; [ ... ])`

Reply:
Done.